src/QMCDrivers/WFOpt/LinearMethod.h

```diff
@@ -36,6 +36,18 @@ class LinearMethod
   void getNonLinearRange(int& first, int& last, const QMCCostFunctionBase& optTarget) const;
 
 public:
+  // Use generalized eigenvalue solver
```

Comment:
Small point, but since the interface to eigensolvers is being expanded, now would be a good time to improve the comments on these function definitions - what is A, B etc. ?